### PR TITLE
Update rlcs.user.js

### PR DESCRIPTION
**Pull request by kretenkobr2**
kretenkobr2 and n1 fixed

### CODE REVIEW

File: rlcs.user.js [hunk 1]
```diff
@@ -763,8 +857,9 @@ ________________________________________________________________________________
 					// Narration Style
 					var msg;
                     var usr = $usr.text();
-                    if (usr == "741456963789852123") { usr = "7-41"; }
-                    //RLClog("usr name for narration "+usr)
+                    if (usr == "741456963789852123") { usr = "7 4 1"; } /* idea: if username is a lot of numbers, call them by the first 3 numbers seperated */
```
Comment (gabewert) [inline rlcs.user.js:860]: Why did you change it from 7-41 to 7 4 1

Comment (Stjerneklar) [inline rlcs.user.js:860]: i felt that it sounds better

Comment (kretenkobr2) [inline rlcs.user.js:860]: I didn't, it was already there, Stjerneklar put it there, the version was wrong, forked on 3.5.2 but 3.8.3(which I actually forked from) already had it
